the_hat_game/players.py

```diff
@@ -43,6 +54,8 @@ def explain(self, word, n_words):
                 timeout=self.timeout,
             )
             word_list = response.json()
+            if not validate_word_list(word_list):
+                raise ValidationError("word_list must be a list of strings")
```

Comment:
Right now if someone will return something different from a word list, we'll raise an error and game will stop. We instead need to log the error and keep the game going.

Reply:
My mistake, everything is correct.